openlibrary/templates/books/edition-sort.html

```diff
@@ -64,7 +64,7 @@
           <!-- Formats are store as a raw string -->
           $book.physical_format.replace('[', '').replace(']','')
           $if book.languages:
-            $:_('in %(languagelist)s', languagelist=commify_list(thingrepr(l) for l in book.languages))
+            $:_('in %(languagelist)s', languagelist=', '.join(l.name for l in book.languages))
```

Comment:
Was there a particular reason you removed `commify_list` here and replaced it with `', '.join` ?

If not, lets put it back since it was probably there for a reason and it used in the other places where we are displaying languages.